install/runners/nunit-tools.wxs

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+	<Product Id="*" 
+             Name="NUnit Runners" 
+             Language="1033" 
+             Version="!(bind.FileVersion.nunit.engine.dll)" 
+             Manufacturer="nunit.org" 
+             UpgradeCode="94E18606-9590-4BA1-B002-258860D7B151">
+        
+        <Package InstallerVersion="200" Compressed="yes" InstallScope="perMachine" />
+        
+        <!-- Include the information that is common to all installs -->
+        <?include ..\common\common.wxi ?>
+
+        <!-- ***************************************************************** -->
+        <!-- **********      Define the NUnit feature tree          ********** -->
+        <!-- ***************************************************************** -->
+        
+        <Feature Id="NUNIT.RUNNERS" 
+                 ConfigurableDirectory="INSTALLDIR" 
+                 Level="1" 
+                 Title="NUnit Runners" 
+                 Display="expand" 
+                 Description="Installs the NUnit engine and tools that allow you to run tests.">
+
+            <ComponentGroupRef Id="NUNIT.COMMON"/>
+
+            <Feature Id="NUNIT.ENGINE"
+                     Level="1"
+                     Absent="disallow"
+                     Title="NUnit Engine"
+                     Description="Installs the NUnit engine">
+                <ComponentGroupRef Id="NUNIT.ENGINE" />
+            </Feature>
+
+            <Feature Id="NUNIT.CONSOLE"
+                     Level="1"
+                     Title="NUnit Console"
+                     Description="Installs the NUnit console runner">
+                <ComponentGroupRef Id="NUNIT.CONSOLE" />
+            </Feature>
+		</Feature>
+	</Product>
+
+    <!-- ***************************************************************** -->
+    <!-- **********       Define Directory Structure            ********** -->
+    <!-- ***************************************************************** -->
+    
+	<Fragment>
+		<Directory Id="TARGETDIR" Name="SourceDir">
+			<Directory Id="ProgramFilesFolder">
+                <Directory Id="INSTALLDIR" Name="NUnit.org">
+                    <Directory Id="TOOLS" Name="tools" />
```

Comment:
I've made the changes based on your other comments, but left this one. I don't think engine, console or tools is right for this directory name. I am hoping it becomes clearer once we get the engine loading issue sorted out. If everything is together, maybe bin for executables and lib for the framework?